src/helpers/strings-en.ts

```diff
@@ -118,6 +118,11 @@
     "select-symptoms": "Select Symptoms",
     "include-treatments": "Include Treatments",
     "select-treatments": "Select Treatments",
+    "new-points-title": "Well Done!",
+    "new-points-text": "You've been awarded points for the following:",
+    "new-points-next-reward-prefix": "You now need ",
+    "new-points-next-reward-suffix": " points to unlock your next reward.",
+    "new-points-done-button-text": "DONE"
```

Comment:
I think for default out of box we should go with "Done" since we don't do all caps for any other buttons

Reply:
Done.